app/gpio/GpioTimerService.js

```diff
@@ -16,12 +16,12 @@ log.setLevel(config.loglevel.default)
 
 export function createGpioTimerService () {
   if (Gpio.accessible) {
-    if (config.gpioHighPriority) {
+    if (config.gpioPriority) {
       // setting top (near-real-time) priority for the Gpio process, as we don't want to miss anything
-      log.debug('setting priority for the Gpio-service to maximum (-20)')
+      log.debug(`Setting priority for the Gpio-service to ${config.gpioPriority}`)
       try {
         // setting priority of current process
-        os.setPriority(-20)
+        os.setPriority(config.gpioPriority)
       } catch (err) {
         log.debug('need root permission to set priority of Gpio-Thread')
       }
```

Comment:
Below on line 37 there is this code:

```js
    sensor.watch((err, value) => {
      if (err) {
        throw err
      }
      const hrDelta = process.hrtime(hrStartTime)
      hrStartTime = process.hrtime()
      const delta = hrDelta[0] + hrDelta[1] / 1e9
      process.send(delta)
    })
  }
```

In this you are re-reading the `hrTime` when you save it to `hrStartTime`. But I believe that could lead to potential (arguably subtle) inaccuracies. 

I would rather read the `hrTime` at the beginning of the function, and save it to a variable and use that all the way. This would ensure consistency. The reason is that Rpi does not have a real interrupt so it could happen that while you run the code in the "interrupt" it does something else and the second `hrTime` read gets executed with a short delay yielding potential inaccuracies. 

Also I believe that process.hrTime([time]) is [legacy code](https://nodejs.org/api/process.html#processhrtimetime) and the following is recommended to be used instead [process.hrtime.bigint()](https://nodejs.org/api/process.html#processhrtimebigint)

Reply:
Hi Abasz, this PR does not introduce any changes to the way the timer works.
It's great that you want to contribute on reviewing this change, but please only comment on things that  are influenced by this PR.
Otherwise an already huge PR like this might become very hard to manage.
If you think that the timer needs improvement, please file an issue for that or you could also create an own PR that a addresses it.

Reply:
I understand your recommendation and I am happy to move this to a new issue. The reason I felt that this should be included here is that it relates to the noise/debounce/impulse detection timing considerations we are discussing [here](https://github.com/laberning/openrowingmonitor/pull/84#discussion_r973748915). Let me know if you still would like me to close this and open an issue instead.

Reply:
May I suggest that @Abasz opens an issue to improve the timing accuracy. I see he has suggested a great GPIO library with an impressive timing resolution, which I am more than happy to test. Also these improvements could increase timing accuracy. If @laberning agrees, we could use that to replace the existing GPIO library and improve the measurement accuracy. As @Abasz suggests, this will probably introduce the need for debounce as well, which we could include in the resulting PR as well. A sneak peak suggests that the library proposed by @Abasz could perform the debounce as well, but I haven't looked at the pro's and cons of that library and its debounce to assess the side effects.

Reply:
Ok. I will close this comment flow and move it to a new issue. My suggestion is that we only move forward with this once the PR is moved to a beta. Accuracy is not bad so I would consider this as a further improvement that should not affect the progress of the PR merging.